.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -9,7 +9,7 @@ body:
     value: |
       Thanks for filling out this bug report.
       - Bugs related to the [AMP](https://amp.dev) format and cache can be reported using the form below.
-      - Bugs related to the [AMP Wordpress Plugin](https://wordpress.org/plugins/accelerated-mobile-pages/) can be reported at the [`amp-wp`](https://github.com/ampproject/amp-wp/issues) repository.
+      - Bugs related to the [AMP WordPress Plugin](https://wordpress.org/plugins/amp/) can be reported at the [`amp-wp`](https://github.com/ampproject/amp-wp/issues) repository, but first consider the [support forum](https://wordpress.org/support/plugin/amp/).
```

Comment:
This will need to be done for the feature request template as well.

Reply:
I took the liberty of pushing a commit that makes this change. I also made the preferred option appear first for better readability. With this, the entire sentence appears on one line for a cleaner look.

![image](https://user-images.githubusercontent.com/26553114/118365464-e484c780-b56a-11eb-9b61-23cf936659ff.png)